tests/test_polar_ft.py

```diff
@@ -0,0 +1,223 @@
+import numpy as np
+import pytest
+
+from aspire.image import Image
+from aspire.operators import PolarFT
+from aspire.utils import gaussian_2d, grid_2d
+from aspire.volume import AsymmetricVolume, CnSymmetricVolume
+
+# ==========
+# Parameters
+# ==========
+
+IMG_SIZES = [
+    64,
+    65,
+]
+DTYPES = [
+    np.float64,
+    np.float32,
+]
+
+RADIAL_MODES = [
+    2,
+    3,
+    4,
+    5,
+    8,
+    9,
+    16,
+    17,
+]
+
+
+# ==================
+# Parameter Fixtures
+# ==================
+
+
+@pytest.fixture(params=DTYPES, ids=lambda x: f"dtype={x}")
+def dtype(request):
+    return request.param
+
+
+@pytest.fixture(params=IMG_SIZES, ids=lambda x: f"img_size={x}")
+def img_size(request):
+    return request.param
+
+
+@pytest.fixture(params=RADIAL_MODES, ids=lambda x: f"radial_mode={x}")
+def radial_mode(request):
+    return request.param
+
+
+# =====================
+# Image and PF Fixtures
+# =====================
+
+
+@pytest.fixture
+def gaussian(img_size, dtype):
+    """Radially symmetric image."""
+    gauss = Image(
+        gaussian_2d(img_size, sigma=(img_size // 10, img_size // 10), dtype=dtype)
+    )
+    pf = pf_transform(gauss)
+
+    return pf
+
+
+@pytest.fixture
+def symmetric_image(img_size, dtype):
+    """Cyclically (C4) symmetric image."""
+    symmetric_vol = CnSymmetricVolume(
+        img_size, C=1, order=4, K=25, seed=10, dtype=dtype
+    ).generate()
+    symmetric_image = symmetric_vol.project(np.eye(3, dtype=dtype))
+    pf = pf_transform(symmetric_image)
+
+    return pf
+
+
+@pytest.fixture
+def asymmetric_image(img_size, dtype):
+    """Asymetric image."""
+    asymmetric_vol = AsymmetricVolume(img_size, C=1, dtype=dtype).generate()
+    asymmetric_image = asymmetric_vol.project(np.eye(3, dtype=dtype))
+    pf = pf_transform(asymmetric_image)
+
+    return asymmetric_image, pf
+
+
+@pytest.fixture
+def radial_mode_image(img_size, dtype, radial_mode):
+    g = grid_2d(img_size, dtype=dtype)
+    image = Image(np.sin(radial_mode * np.pi * g["r"]))
+    pf = pf_transform(image)
+
+    return pf, radial_mode
+
+
+# Helper function
+def pf_transform(image):
+    """Take polar Fourier transform of image."""
+    img_size = image.resolution
+    nrad = img_size // 2
+    ntheta = 360
+    pft = PolarFT(img_size, nrad=nrad, ntheta=ntheta, dtype=image.dtype)
+    pf = pft.transform(image)
+    pf = pf.reshape(ntheta // 2, nrad)
```

Comment:
Think we missed removing this line. `pf` is supposed to be this shape already right?

Reply:
Ah, good catch. Thanks!

Reply:
Fixed it up. Thanks